docker-compose-shared.yml

```diff
@@ -0,0 +1,17 @@
+# Docker Compose over ride file for FreeCodeCamp
+#
+# Run with:
+#   docker-compose -f docker-compose.yml -f docker-compose-shared.yml up
+#
+version: "2"
+services:
+    db:
+        networks:
+            - shared
+    freecodecamp:
+        networks:
+            - shared
+
+networks: # Used by by other projects like open-api
```

Comment:
Creates a network that can be used by other projects wishing to use these resources

Reply:
![I like it a lot](https://media.giphy.com/media/Jlt69Ka6SwtH2/giphy.gif)